ftplugin/latex-suite/texviewer.vim

```diff
@@ -733,6 +743,10 @@ if has('python') && Tex_GetVarValue('Tex_UsePython')
 	python import sys, re
 	exec "python sys.path += [r'". s:path . "']"
 	python import outline
+elseif has('python3') && Tex_GetVarValue('Tex_UsePython')
+	python3 import sys, re
+	exec "python3 sys.path += [r'". s:path . "']"
+	python3 import outline
```

Comment:
I like that you introduce the pythoncmd variable in line 13 and 16. But there are several lines where you do not use it. Would you please use it in all places?